zerver/actions/streams.py

```diff
@@ -1176,7 +1176,7 @@ def send_change_stream_permission_notification(
     with override_language(stream.realm.default_language):
         notification_string = _(
             "{user} changed the [access permissions](/help/stream-permissions) "
```

Comment:
Small observation: We should probably be sure to either make the help links containing `stream` be passed as parameters (ideal) or switched to "channel" before doing a push for translators to work on updating strings.

Reply:
Quick audit of the translated strings for this shows these messages and the onboarding messages (which are being revised/changed). I'll update these to be passed as parameters.

Reply:
Updated these strings to have a parameter for the help center link.